src/lib/redux/cache.js

```diff
@@ -7,7 +7,7 @@ const HOUR_IN_MS = 3600000;
 const SERIALIZE_THROTTLE = 500;
 const MAX_AGE = 30 * DAY_IN_HOURS * HOUR_IN_MS;
 const STORAGE_KEY = 'REDUX_STATE';
-const SERIALIZED_STATE_VERSION = 3;
+const SERIALIZED_STATE_VERSION = 4;
```

Comment:
I think to avoid this, we should probably avoid storing the endpoints discovery on the localstorage, We con do that by dropping the schema parameter from the `createReducer` call on the endpoints reducer.

But I thought it's a good idea to keep in on production, maybe we could change this parameter depending on the value of `process.env.NODE_ENV`